### DISCUSSION TITLE
Tell spammers that a mail address no longer exists

### DISCUSSION THREAD

**Discussion by Satlinker** [simple-login/app#783] [Feature Requests]
You can stop receiving mails from a specific sender, but the mails are still accepted from simplelogin and discarded. It would be better to tell a spammer that the mail address is no longer valid. So he should get an answer that his mail is blocked, undeliverable, receiver unknown or similar.

**Comment by JoeBoll** [comment]
In my personal case I often disable aliases for services I rarely use as to avoid their excessive newsletters, I then turn them on if I need to reset my password or email support. So in my own case, SimpleLogin telling them my email address is a bounce would reduce functionality for me.

**Comment by Satlinker** [comment]
It makes sense to disable (pause) newsletters like it's implemented just now. Alternatively it should be possible to block an address, otherwise a spammer doesn't stop spamming because he doesn't know that his mails are discarded. Both should be possible.

**Reply by JoeBoll** [reply]
I think that would probably be good for when blocking a sender or when deleting an alias.

**Comment by Satlinker** [comment]
An even better idea would be to temporarily block all contacts that are not in the actual contact list. This would stop spammers who change their sender address every time.
Ideally all members of a domain like ...@company1.com should be switchable to "unblocked". This would be a possibility to create aliases e.g. only for a specific customer. A problem is, that you get noreplay@ebay.com, ebay@ebay.com, postmaster@ebay.com.... so a way to generally unblock @ebay.com would be great. That makes no sense for private mail domains like @gmail.com. I even have ebay@ebay.com multiple times in the contacts list, it seems that simple-login doesn't recognize it.

**Comment by Satlinker** [comment]
Now I think I have the best solution:
Disable an alias or a single contact should be selectable between discard and reject.
Disable an alias should be selectable between disabling "all" or "all new contacts" or "new contacts from unknown domains".

**Comment by Satlinker** [comment]
To bring this thing up again: blocking a sender means for me to tell him that the address is no longer available. Just discard the mails can be done by a simple blacklist from every mail client.

**Comment by jnv** [comment]
Under Dashboard > Settings there is option to reject emails for disabled aliases and blocked senders. I think this addresses your feature request?

![](https://user-images.githubusercontent.com/616767/173030461-e2f1d3b8-8123-4d80-8c3f-f73bfe175ab5.png)

I recommend opening a new feature request for other ideas, for example to block "new contacts from unknown domains".

**Comment by Satlinker** [comment]
Great, that's it.
